plan/planbuilder.go

```diff
@@ -488,11 +488,24 @@ func (b *planBuilder) buildAnalyzeIndex(as *ast.AnalyzeTableStmt) Plan {
 	return p
 }
 
+func (b *planBuilder) buildAnalyzeAllIndex(as *ast.AnalyzeTableStmt) Plan {
+	p := &Analyze{}
+	tblInfo := as.TableNames[0].TableInfo
+	for _, idx := range tblInfo.Indices {
```

Comment:
If we use primary key as handle, it will not appear in tblInfo.Indices.

Reply:
Yes, it is expected.